libnetwork/internal/netiputils/netiputils.go

```diff
@@ -1,4 +1,4 @@
-package ipam
+package netiputils
```

Comment:
```suggestion
package netiputil
```
Why use many chars when few do trick?

Reply:
k